content/en/docs/reference/command-line-tools-reference/feature-gates/dynamic-resource-allocation.md

```diff
@@ -8,7 +8,8 @@ _build:
 stages:
   - stage: alpha
     defaultValue: false
-    fromVersion: "1.26"
+    fromVersion: "1.30"
```

Comment:
According to the DRA [kep.yaml ](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/kep.yaml)and this is to the `dev-1.31` branch and I see DRA made 1.31 code freeze in this [comment](https://github.com/kubernetes/enhancements/issues/4381#issuecomment-2245765666), should this be 1.31?